Index: SmallObj.cpp
===================================================================
--- SmallObj.cpp	(revision 1191)
+++ SmallObj.cpp	(working copy)
@@ -1197,13 +1197,9 @@
 
     if ( ( nullptr == place ) && doThrow )
     {
-#ifdef _MSC_VER
-        throw std::bad_alloc( "could not allocate small object" );
-#else
         // GCC did not like a literal string passed to std::bad_alloc.
         // so just throw the default-constructed exception.
         throw std::bad_alloc();
-#endif
     }
     return place;
 }
